CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ option(MI_USE_CXX           "Use the C++ compiler to compile the library" OFF)
 option(MI_SEE_ASM           "Generate assembly files" OFF)
 option(MI_LOCAL_DYNAMIC_TLS "Use slightly slower, dlopen-compatible TLS mechanism (Unix)" OFF)
 option(MI_BUILD_TESTS       "Build test executables" ON)
+option(MI_USER_CLEANUP      "Enable mi_register_user_cleanup functionality" OFF)
```

Comment:
How about changing `mi_register_user_cleanup` to something user-readable? Plain text in regular English might be better.